pkg/kudoctl/cmd/get.go

```diff
@@ -2,18 +2,34 @@ package cmd
 
 import (
 	"github.com/kudobuilder/kudo/pkg/kudoctl/cmd/get"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
-// newGetCmd creates a new command that lists instances
+// newGetCmd creates a command that lists the instances in the cluster
 func newGetCmd() *cobra.Command {
-	newCmd := &cobra.Command{
-		Use:   "get",
-		Short: "-> Show all available instances.",
-		Long:  `The get command has subcommands to show all available instances.`,
+	options := get.DefaultOptions
+	var parameters []string
+	getCmd := &cobra.Command{
+		Use:   "get instances",
+		Short: "Gets all available instances.",
+		Long: `
+	# Get all available instances
+	kudoctl get instances`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Prior to command execution we parse and validate passed parameters
+			var err error
+			options.Parameters, err = getParameterMap(parameters)
```

Comment:
we don't accept any parameters here, is this also copy-paste error?

Reply:
Not a copy paste error. Imo it should pave the way for more functionality further down the road.

Reply:
I see your point though. I just have this wish we can make it look all more homogenous and use same patterns in `install`, `get`, `plan` and so on.